docs/api/desktop-capturer.md

```diff
@@ -100,6 +99,20 @@ Returns `Promise<DesktopCapturerSource[]>` - Resolves with an array of [`Desktop
 [`navigator.mediaDevices.getUserMedia`]: https://developer.mozilla.org/en/docs/Web/API/MediaDevices/getUserMedia
 [`systemPreferences.getMediaAccessStatus`]: system-preferences.md#systempreferencesgetmediaaccessstatusmediatype-windows-macos
 
+## Loopback Audio on macOS 13/14/15
+
+On macOS 13 and macOS 14, `navigator.mediaDevices.getDisplayMedia` may be used to capture loopback audio if the [`setDisplayMediaRequestHandler`](./session.md#sessetdisplaymediarequesthandlerhandler-opts) callback is returned with `audio: 'loopback'`.
+
+For macOS 15+, loopback audio is only supported if the following experimental command line switches are appended:
```

Comment:
I would say "only works if". It's an experimental feature that implies it isn't supported yet.